libretroshare/src/serialiser/rstypeserializer.cc

```diff
@@ -328,6 +335,49 @@ bool RsTypeSerializer::from_JSON( const std::string& memberName,
 	return ret;
 }
 
+template<> /*static*/
+uint32_t RsTypeSerializer::serial_size(const double&)
```

Comment:
why is this needed? Either we try to serialize doubles, then we need this, or we don't, then we dont need this either. Why would JSon need doubles where the binary API doesn't? Also if necessary, serializing doubles is quite possible. I did it for float32 already I think.

Reply:
We need this for JSON because some parameter type of some method exposed in the JSON API use double, binary serialization is not needed ATM because that kind of type is never exchanged between retroshare instances, I don't know the math behind floating point binary serialization so i haven't implemented it as is not needed ATM, but if you want to contribute that you are welcome to do that ;)